custom_metrics/third-parties.js

```diff
@@ -0,0 +1,34 @@
+[third-parties]
```

Comment:
That's how the block definition works for WebPageTest when submitting through the UI or API but not for permanent ones installed on the server. Those should just be scripts with one script per metric.

i.e. yes, it should be removed.